extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -174,17 +201,17 @@ private static function isProduction() {
 	}
 
 	/**
-	 * @param $metaDataFromService is actual Data returned by Liftigniter Metadata Service
+	 * @param Item $metaDataFromService is actual Data returned by Liftigniter Metadata Service
 	 *
 	 * @return bool
 	 */
-	private static function shoudlNoIndex( $metaDataFromService ) {
+	private static function shouldNoIndex( $metaDataFromService ) {
 		global $wgDisableShowInRecirculation;
 
 		return self::isPrivateOrNotProduction() ||
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)

[project-id]: wikia:app